src/javascript/app_2/Modules/Reports/Helpers/profit-loss.js

```diff
@@ -0,0 +1 @@
+export const isProfitOrLoss = value => +value.replace(/,/g, '') >= 0 ? 'profit' : 'loss';
```

Comment:
if it is 0 is it profit?

Reply:
In P/L terms since you're not suffering from any loss it's a profit.

Reply:
Btw should we not use `is` in the name since then the function might be mistaken for returning a boolean? and instead call it something like `profitOrLoss`

Reply:
agreed @OskarAhl , good spot. Missed the part where it's returning a `string` not a `bool`. Maybe `getProfitOrLoss` instead?